src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/ChunkContextToolbar.ui.xml

```diff
@@ -41,9 +41,40 @@
       box-shadow: 0 0 3px rgb(160, 160, 160);
    }
 
+   .chunkType
+   {
+      display: inline-block;
+      color: #C7C7C7;
```

Comment:
It actually looks pretty descent in all themes, but shoot me bug if you spot something weird, so far so good. CC @dfalty